.github/workflows/frontend.yml

```diff
@@ -93,3 +93,5 @@ jobs:
         args: deploy --only hosting
       env:
         FIREBASE_TOKEN: ${{ secrets.FIREBASE_SERVICE_ACCOUNT_SUPREME_ERP }}
+        PROJECT_PATH: './frontend'
```

Comment:
I don't think this line is necessary, you can test the deployment by commenting out the if statement in the deploy phase

Reply:
This was recommended by the GitHub Action for Firebase documentation.

Reply:
The firebase deployment doesn't use our "repo". We save the files we need in memory than we download it to deploy with firebase. Right now it's set up to download the build folder and the `firebase.json` and `firebaserc`. I think it downloads those files to root